source/settings.txt

```diff
@@ -3,3 +3,69 @@
 ============================
 MongoDB for VS Code Settings
 ============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Configure Settings
+------------------
+
+To configure MongoDB for VS code settings:
+
+.. include:: /includes/steps/configure-settings.rst
+
+Settings
+--------
+
+The following table lists the available settings for 
+MongoDB for VS Code:
+
+.. list-table::
+   :header-rows: 1
+   
+   * - Setting
+     - Description
+
+   * - :guilabel:`Shell`
+     - MongoDB shell to use. Defaults to ``mongo``.
+
+   * - :guilabel:`Show`
+     - If checked, shows the MongoDB view in the Visual Studio Code 
+       left navigation panel.
+
+   * - :guilabel:`Max Number of Problems`
+     - The maximum number of problems MongoDB for VS Code will generate 
+       in the Visual Studio Code :guilabel:`Problems` view.
+
+   * - :guilabel:`Confirm Run All`
+     - If checked, shows a confirmation message before running 
+       commands in a playground.
+
+   * - :guilabel:`Connection Saving: Default Connection Saving Location`
+     - Sets if and where new connections are saved. Defaults to 
+       ``Global``.
```

Comment:
Interesting, mine defaults to `Workspace`.

Reply:
I believe it's Global, restarted mine to double check and looked around in the code a bit, where I saw "// Update something in the storage. Defaults to global storage (not workspace)"
https://github.com/mongodb-js/vscode/blob/master/src/storage/storageController.ts